### PR TITLE
term: "metricS history" (file), et al. [DVCLive]

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Follow up to #3207

Includes other misc. edits.

### CODE REVIEW

File: content/docs/dvclive/api-reference/live/log_image.md [hunk 1]
```diff
@@ -64,9 +64,9 @@ $ tree
 
 ## Parameters
 
-- `name` - Name of the output file.
+- `name` - name of the output file
```
Comment (jorgeorpinel) [inline content/docs/dvclive/api-reference/live/log_image.md:67]: @daavoo what "output file", can we be more specific as to what this does?

File: content/docs/dvclive/api-reference/live/log_plot.md [hunk 2]
```diff
@@ -201,15 +201,15 @@ $ dvc plots show 'dvclive/plots/roc.json' \
 
 ## Parameters
 
-- `name` - One of the supported plots.
+- `name` - a supported plot
```
Comment (jorgeorpinel) [inline content/docs/dvclive/api-reference/live/log_plot.md:204]:
```suggestion
- `name` - a [supported plot type](#supported-plots)
```

File: content/docs/dvclive/api-reference/live/log_plot.md [hunk 3]
```diff
@@ -201,15 +201,15 @@ $ dvc plots show 'dvclive/plots/roc.json' \
 
 ## Parameters
 
-- `name` - One of the supported plots.
+- `name` - a [supported plot type](#supported-plots)
```
Comment (jorgeorpinel) [inline content/docs/dvclive/api-reference/live/log_plot.md:204]: Hi @daavoo what's the default? Thanks

**Review by iesahin — APPROVED** [review]
Looks good after @daavoo 's review. Thanks Jorge.

File: content/docs/dvclive/api-reference/live/log_image.md (hunk 1 above)
Comment (daavoo) [inline content/docs/dvclive/api-reference/live/log_image.md:67]:
```suggestion
- `name` - name of the output file where the image will be saved
```

File: content/docs/dvclive/api-reference/live/log_plot.md (hunk 3 above)
Comment (daavoo) [reply content/docs/dvclive/api-reference/live/log_plot.md:204]: There is no default. It's mandatory

**Pull request merged**